internal/mode/static/manager.go

```diff
@@ -212,6 +213,8 @@ func StartManager(cfg config.Config) error {
 			ngxPlusClient,
 			ngxruntimeCollector,
 			cfg.Logger.WithName("nginxRuntimeManager"),
+			&ngxruntime.ProcessHandlerImpl{},
+			ngxruntime.NewVerifyClient(1*time.Second),
```

Comment:
This timeout should be the same as `nginxReloadTimeout` in the runtime manager. Let's export and use it.

Reply:
sure